db/migrate/20160408012345_create_pg_search_documents.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePgSearchDocuments < ActiveRecord::Migration
+  def self.up
```

Comment:
This migration is the one generated by pg search plus some modifications. It seems like they like the old syntax for many ruby/rails things. I can change if wanted but not important imo